docs/code/getting-started.md

```diff
@@ -675,4 +675,123 @@ Let’s recap what we learned in this section:
+
+	Gtk.Image image = new Gtk.Image.from_icon_name ("application-default-icon", Gtk.IconSize.DIALOG);
+	notification.set_icon (image.gicon);
+
+And that's it! Compile your app again, and press that "Send" button. As you can see, the notification now has an icon! Using this method, by changing the first string you can set the icon to anyone you wish! You can use ```gtk3-icon-browser``` to see what system icons are available for you to use.
+
+### Replace {#replace}
+We now know how to send a notification, <!--but now the user has interacted with your app. How do we remove it? --> but what if you need to update it with new information? Well we can easily do it thanks to the notification ID. This ID can be any string you wish used for you to be able to identify which notification is which from the code. For now we just named it `"notify-test"`. When you start coding your own app, notifications IDs should be your app's name.
+
+Let's do the replace button. This button will replace the current notification with one with different information. Let's create a new button for it, and let's add it to the grid:
+
+	var update_button = new Gtk.Button.with_label (_("Update"));
+	grid.add (update_button);
+
+	update_button.clicked.connect (() => {
+		var notification = new Notification (_("Hello Again"));
+		notification.set_body (_("This is my second Notification!"));
+
+		Gtk.Image image = new Gtk.Image.from_icon_name ("application-default-icon", Gtk.IconSize.DIALOG);
+		notification.set_icon (image.gicon);
+
+		this.send_notification ("notify-test", notification);
+	});
+<!--
+Now, let's do the withdraw button:
+
+	var withdraw_button = new Gtk.Button.with_label (_("Withdraw"));
+	grid.add (withdraw_button);
+
+	withdraw_button.clicked.connect (() => {
+		this.withdraw_notification ("notify-test");
+	});
+-->
+And that's it! Very easy right? Now let's compile and run your app again. Click on the buttons, first on the Show, then Update, and later on the Withdraw. 
+
+###Priority  {#priority}
+Notifications also have something called priority. In elementary OS, when a notification is set as "urgent" it will stay on the screen until either the user interacts with it, or you withdraw it. Let's make our notification urgent! To do so all you have to do is add the following line before the `this.send_notification ()` function
+
+	notification.set_priority (NotificationPriority.URGENT);
+
+Urgent notifications though should really only be used on the most extreme of cases. For example, here in elementary we use them for telling the user that his battery is about to run out, or when a printing job failed. There are also [other notification priorities](http://valadoc.org/#!api=gio-2.0/GLib.NotificationPriority). In elementary OS Loki, notifications with priority set to LOW are skipped from the notifications indicator.
```

Comment:
Probably should be consistent about making the enums all caps and as code like "`URGENT` notifications should only..." and "with priority set to `LOW`"